src/config/process.ts

```diff
@@ -116,3 +116,10 @@ export const getLoopConfig = () => {
     lnd2LoopMacaroon: process.env.LND2_LOOP_MACAROON as Macaroon,
   }
 }
+
+export const getKratosMasterPhonePassword = () => {
```

Comment:
I understand the reason for this - just wondering wether we shouldn't be generating a unique PW per user instead of using the same one everywhere. If this gets leaked what are the repercussions? Will this go away once the symmetric JWT is gone?
We can have self-serviced on boarding from mobile talking directly to kratos (like in web wallet) shouldn't we be doing that?

Reply:
> I understand the reason for this - just wondering wether we shouldn't be generating a unique PW per user instead of using the same one everywhere.

there is no reason to. it's more state to manage with no clear benefits

> If this gets leaked what are the repercussions?

it should be kept in the backend. it's like having a private key, or the prior secret for signing the JWT. yes if this get leaked this means a breach happen but in that case an attacker would not just take this password but more likely private keys. 

> We can have self-serviced on boarding from mobile talking directly to kratos (like in web wallet) shouldn't we be doing that?

we can't for phone. it will always (or at least for the time being) be behind some logic in our api. the main reasons:
1/ there is no phone support in kratos
2/ there is no captcha support in kratos (and this one will likely never be resolved if there is no open source catcha solution)

unlike with email, phone __needs__ to be protected behind a captcha, otherwise there will "DDOS text" attacks